.bmad-core/templates/project-brief-tmpl.yaml

```
@@ -0,0 +1,221 @@
+
+  - id: next-steps
+    title: Next Steps
+    sections:
+      - id: immediate-actions
+        title: Immediate Actions
+        type: numbered-list
+        template: "{{action_item}}"
+      - id: pm-handoff
+        title: PM Handoff
+        content: |
+          This Project Brief provides the full context for {{project_name}}. Please start in 'PRD Generation Mode', review the brief thoroughly to work with the user to create the PRD section by section as the template indicates, asking for any necessary clarification or suggesting improvements.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)